web/docs/tutorial/02-project-structure.md

```diff
@@ -104,17 +132,6 @@ page MainPage {
 </TabItem>
 </Tabs>
 
-<ShowForTs>
-
-:::caution Using TypeScript
-The default project uses JavaScript. To use TypeScript, you must rename the file
```

Comment:
Removed the admonition (as per the style guide), and moved to the top of the document.